fix gh build action error 5

@claude i am still running into quality check errors in the  when i run the .gitbub/workflows/build-app.yml it is failing with this error:

```
6s
Run npm run typecheck

> openwebui-config-tool@1.0.0 typecheck
> tsc --noEmit

Error: src/utils/validation.ts(51,19): error TS2693: 'OpenWebUIConfig' only refers to a type, but is being used as a value here.
Error: src/utils/validation.ts(119,20): error TS2693: 'OpenWebUIConfig' only refers to a type, but is being used as a value here.
Error: Process completed with exit code 2.
```

and the gh workflow indicates: 
```
<div class="mt-1 mb-3 px-3" style="box-sizing: border-box; margin-top: 4px !important; margin-bottom: 16px !important; padding-right: 16px !important; padding-left: 16px !important; color: rgb(255, 255, 255); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="mx-0 mx-md-1" style="box-sizing: border-box; margin-right: 0px !important; margin-left: 0px !important;"><h2 class="text-bold h4" style="box-sizing: border-box; margin-top: 0px; margin-bottom: 0px; font-size: 16px !important; font-weight: 600 !important;">Annotations</h2><div class="color-fg-muted text-small" style="box-sizing: border-box; color: rgb(183, 189, 200) !important; font-size: 12px !important;">3 errors</div></div></div>
build-and-commitProcess completed with exit code 2.
--
build-and-commit: src/utils/validation.ts#L119'OpenWebUIConfig' only refers to a type, but is being used as a value here.
build-and-commit: src/utils/validation.ts#L51'OpenWebUIConfig' only refers to a type, but is being used as a value here.

Annotations
3 errors
[build-and-commit](https://github.com/leger-labs/openwebui-config.com/actions/runs/17316244395/job/49159552399#step:6:14)
Process completed with exit code 2.
[build-and-commit: src/utils/validation.ts#L119](https://github.com/leger-labs/openwebui-config.com/commit/01b92119df66d524a5953bab98293f060168d66b#annotation_38371095544)
'OpenWebUIConfig' only refers to a type, but is being used as a value here.
[build-and-commit: src/utils/validation.ts#L51](https://github.com/leger-labs/openwebui-config.com/commit/01b92119df66d524a5953bab98293f060168d66b#annotation_38371095547)
'OpenWebUIConfig' only refers to a type, but is being used as a value here.
```

this may be due to poor interfacing between the generated-*.ts code that maps the single source of truth /schema .json file to the /src using /scripts.
if that is the case then you are to apply the fix at the /scripts/ level not the auto-generated code in /src/schemas

resolve this issue

--- a/src/utils/validation.ts
+++ b/src/utils/validation.ts
@@ -1,6 +1,6 @@
 import { z } from 'zod'
 import type { ConfigData, ValidationError, EnvVariable } from '@/types'
-import { OpenWebUIConfig } from '@/schemas/generated-schemas'
+import { OpenWebUIConfigSchema } from '@/schemas/generated-schemas'
 
 // Validation result type
 export interface ValidationResult {
@@ -48,7 +48,7 @@ export function validateField(
 // Get field schema from OpenWebUIConfig
 function getFieldSchema(fieldName: string): z.ZodSchema | null {
   try {
-    const shape = OpenWebUIConfig.shape as Record<string, z.ZodSchema>
+    const shape = OpenWebUIConfigSchema.shape as Record<string, z.ZodSchema>
     return shape[fieldName] || null
   } catch {
     return null
@@ -116,7 +116,7 @@ export function validateConfig(data: ConfigData): ValidationResult {
 
   try {
     // Validate against full OpenWebUI schema
-    const result = OpenWebUIConfig.safeParse(data)
+    const result = OpenWebUIConfigSchema.safeParse(data)
     
     if (!result.success) {
       result.error.errors.forEach((error: any) => {

